lib/rspec/buildkite/analytics/uploader.rb

```diff
@@ -130,13 +130,21 @@ def self.configure
               puts "Error communicating with the server: #{e.message}"
             end
 
-            if response.is_a?(Net::HTTPSuccess)
+            case response.code
+            when "401"
+              puts "Invalid Suite API key. Please double check your Suite API key."
+            when "200", "201"
```

Comment:
Just wondering why we're accounting for both a 200 and a 201 return here? Like does the server not consistently return the same status code for this initial auth request?? And if it doesn't, should we be accounting for more values in the 2XX range then?

Reply:
I was trying to be generous, but we always return 200. Fixed in c18fdf21cb6350fa809ca40749f1e97ecd7fec06. Thanks!